.github/workflows/ci.yml

```diff
@@ -183,12 +183,21 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - name: Test on node.js ${{ matrix.node-version }}
+        if: matrix.node-version == '8'
         # Hack: --color has supports-color@5 returned true for GitHub CI
         # Remove once `chalk` is bumped to 4.0.
 
         # Todo(Babel 8): Jest execution path is hardcoded because Yarn 2 does not support node 6
         run: |
-          BABEL_ENV=test node ./node_modules/.bin/jest --ci --color
+          BABEL_ENV=test FUZZ=false node --max-old-space-size=4096 ./node_modules/.bin/jest --ci --color
```

Comment:
Done: https://github.com/babel/babel/pull/14201/commits/ffb642e15e3163f14937783ce19a083e58fc74e9

Reply:
With regard to the memory stuff, I had originally put it in there because in a very early version it was running out of memory, but no longer. However, it does help with performance generally, since the older versions of node have a bug where they can't automatically address more than 2GB (as opposed to older versions which can without any special flag), and the fact that the tests are written in a way where before any single test is run node is basically taken to the very threshold of its memory limit, because it starts by reading every single test file, before even deciding whether to skip tests, etc. So basically it puts the entirety of all the tests into memory and keeps them there throughout most of the run. Hence any additional memory it can get helps tremendously (I think it shaves a minute of Node 6 runtime -- regardless of fuzz testing or not). This is something I'm planning to attack in my next PR: only read the files when its actually time to run the tests, such that with TEST_GREP thousands of files don't get read at all.